base/applications/charmap/map.c

```diff
@@ -558,9 +558,9 @@ MapWndProc(HWND hwnd,
     {
         case WM_CREATE:
         {
-            if (!OnCreate(infoPtr,
-                          hwnd,
-                          ((LPCREATESTRUCTW)lParam)->hwndParent))
+            if (!MapOnCreate(infoPtr,
+                             hwnd,
+                             ((LPCREATESTRUCTW)lParam)->hwndParent))
```

Comment:
lgtm. (You might consider also renaming the other one, up to you).

Reply:
Will do.